Bottom: 7f65405d39cff8c832fa9d06e57fd40af6a9098e
Top:    7169998974ca4cacb88921c1cc3869a2c87c20f2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-07 10:02:31 +0000

Refresh of add-support-for-a-poured-objec

---

diff --git a/src/draw.c b/src/draw.c
index 6d668e5..56b99e9 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
